ivadomed/utils.py

```diff
@@ -303,18 +280,15 @@ def get_arguments(parser, args):
     return args
 
 
-def __get_commit(path_to_git_folder=None):
-    """Get GIT ivadomed commit.
+def __get_commit(path_to_git_folder):
+    """Get a git commit.
```

Comment:
This function is better implemented as

```
return subprocess.check_output(["git","describe","--always","--dirty"]))
```

that's the standard form most projects use. A dirty repo is reported as "-dirty", instead of with a "*", but otherwise it's the same, and can be done in one line. But that can be done in a follow-up.